examples/embedded_resource_manager_ex.py

```diff
@@ -0,0 +1,20 @@
+import sys, os
+sys.path.append(os.getcwd())
+from locust_plugins.embedded_resource_manager import HttpUserWithResources
+from locust import task, events
+import time
```

Comment:
unused, right?

Reply:
You can see I have played around with a few more things before ending up here... 

I'll remove it